.travis.yml

```diff
@@ -1,11 +1,37 @@
-language: python
+language: node_js
 python:
-  - "2.7"
-before_script:
-  - psql -c 'create database oneanddone;' -U postgres
-  - cp oneanddone/settings/local.py-travis oneanddone/settings/local.py
+- "2.7"
+node_js:
+- "0.10"
+addons:
+  artifacts:
+    paths:
+    - results
+    permissions: public-read
+    bucket: fireplace-tests
```

Comment:
Good catch. I need to set up a new bucket for these tests. Also, the secure variables in here will only work for my fork right now, so I should update those to work for the upstream repo.